internal/pkg/term/selector/selector.go

```diff
@@ -701,6 +706,51 @@ func (s *WorkspaceSelector) Workload(msg, help string) (wl string, err error) {
 	return selectedWlName, nil
 }
 
+// WSEnvironment fetches all environments belong to the app in the workspace and prompts the user to select one.
+func (s *WorkspaceSelector) WSEnvironment(msg, help string) (wl string, err error) {
```

Comment:
Can we rename this to the following? since the method is for a `WorkspaceSelector` it feels a bit redundant.
```suggestion
func (s *WorkspaceSelector) Environment(msg, help string) (string, error) {
```

Reply:
This is because `WorkspaceSelector` contains `ConfigSelector` which contains `AppEnvSelector` which has a method named `Environment`. Renaming `WSEnv` to `Env` overshadows the `Env` method of `AppEnvSelector`, and will break some other `cli` commands which uses `WorkspaceSelector` to invoke `Environment` method "inherited" from the `AppEnvSelector`... I wanted to refactor it but it'd remove the method "inheritance" between the bunch of selectors.

Reply:
☹️ dang, we shouldn't have embedded the *ConfigSelector to the WorkspaceSelector.

Well - maybe this is a scenario where the name of the type is poor?
What if we had a refactor where we rename `WorkspaceSelector -> LocalWorkloadsSelector` and then have a separate type that's `WorkspaceSelector` that's limited to strictly choosing stuff from a workspace

Reply:
Done!